packages/sdk-middleware-http/test/http.spec.js

```diff
@@ -458,7 +494,60 @@ describe('Http', () => {
       const httpOptions = {
         host: testHost,
         includeOriginalRequest: true,
-        maskSensitiveHeaderData: true,
+      }
+      const httpMiddleware = createHttpMiddleware(httpOptions)
+      nock(testHost)
```

Comment:
why are we `nocking`:ing these repeatedly over putting it into a `beforeEach`?

Reply:
Good question. I honestly didn't think of refactoring anything but just used the same test template for this. Could make another PR and changing that at another point, perhaps.